app/controllers/songs_controller.rb

```diff
@@ -31,6 +32,16 @@ def index
     end
   end
 
+  def show
+    respond_to do |format|
+      @song = Song.find(params[:id])
+      song = @song
```

Comment:
`song` doesn't seem to be used again. And why do we need to make `@song` an instance variable if we are rendering it as JSON directly and not going into a erb view?